src/foam/core/Validation.js

```diff
@@ -37,6 +37,10 @@ foam.CLASS({
         };
       }
     },
+    {
+      class: 'String',
+      name: 'errorMessage'
```

Comment:
Please add documentation to explain how this could be used with validation and documentation on errorString would be useful as well.